src/test/run-pass/param_attrs.rs

```diff
@@ -0,0 +1,28 @@
+// This test ensures we can attach attributes to the formals in all
```

Comment:
@petrochenkov I think we want more tests here that cover at least some of the cases mentioned in the RFC.